.github/workflows/build-live-oe-images-php8.3-noble.yml

```diff
@@ -0,0 +1,277 @@
+          context: ./Web-Live
+          push: false
+          build-args: |-
+            BUILD_BRANCH=${{ env.GITHUB_REF_NAME }}
+            MODULES=${{ matrix.modules }}
+            PHP_VERSION=${{ matrix.php-version }}
+            OS_VERSION=${{ matrix.os-version }}
+            OE_VERSION=${{ steps.meta-live.outputs.slug }}${{ env.IMAGE_SUFFIX }}
+          tags: ${{ steps.meta-live.outputs.tags }}
+      # Manager
+      - name: Build Manager image
+        uses: docker/build-push-action@v5
+        with:
+          ssh: |
+              default=${{ env.SSH_AUTH_SOCK }}
+          context: ./Manager
+          push: false
+          build-args: | # Note that OE_VERSION must use a 'slug' format, not GITHUB_REF_NAME, becuse /'s get converted to -'s
+            BUILD_BRANCH=${{ env.GITHUB_REF_NAME }}
+            MODULES=${{ matrix.modules }}
+            OE_VERSION=${{ steps.meta-dbboot.outputs.slug }}${{ env.IMAGE_SUFFIX }}
+            BASEIMAGE=${{ env.DOCKER_ORG }}/${{ env.DOCKER_WEB_IMAGE }}
+            PHP_VERSION=${{ matrix.php-version }}
+            OS_VERSION=${{ matrix.os-version }}
+          tags: ${{ steps.meta-dbboot.outputs.tags }}
+          
+      # DEBUG
+        # (Debug images contain the same code as the live image, but include additional debugging tools and run in dev mode)
+      - name: Build DEBUG image
+        uses: docker/build-push-action@v5
+        with:
+          ssh: |
+              default=${{ env.SSH_AUTH_SOCK }}
+          context: ./Web-Dev
+          push: false
+          # Note that BASE_IMAGE_TAG must use use a 'slug' format, not GITHUB_REF_NAME, becuse /'s get converted to -'s
+          # PROD_DEBUG=true tells the dockerfile to build a debug image rather than a blank development image
+          build-args: | 
+            BUILD_BRANCH=${{ env.GITHUB_REF_NAME }}
+            BASE_IMAGE_NAME=${{ env.DOCKER_ORG }}/${{ env.DOCKER_WEB_IMAGE }}
+            BASE_IMAGE_TAG=${{ steps.meta-debug.outputs.slug }}${{ env.IMAGE_SUFFIX }}
+            OE_VERSION=debug-${{ steps.meta-debug.outputs.slug }}${{ env.IMAGE_SUFFIX }}
+            PHP_VERSION=${{ matrix.php-version }}
+            OS_VERSION=${{ matrix.os-version }}
+            CYPRESS_VERSION=
+            PROD_DEBUG=TRUE
+          tags: ${{ steps.meta-debug.outputs.tags }}
+          
+
+######################################## 
+# Push final images to docker registry #
+########################################
+
+      # LIVE
+      - name: Push Live image
+        uses: docker/build-push-action@v5
+        with:
+          ssh: |
+              default=${{ env.SSH_AUTH_SOCK }}
+          context: ./Web-Live
+          push: true
+          build-args: |-
+            BUILD_BRANCH=${{ env.GITHUB_REF_NAME }}
+            MODULES=${{ matrix.modules }}
+            PHP_VERSION=${{ matrix.php-version }}
+            OS_VERSION=${{ matrix.os-version }}
+            OE_VERSION=${{ steps.meta-live.outputs.slug }}${{ env.IMAGE_SUFFIX }}
```

Comment:
This is where we need to be careful with how the base images are tagged in OEImageBuilder, as the dockerfiles are relying on a specific tag format. Things will go very wrong if the required tag isn't available for the base, and the same tag isn't available for the built live image, etc (as the later images in the chain rely on the same tag)

I haven't finished reviewing the builder changes yet - but hpoefully what you've done will be fine...